frontend/src/utils/YupSchema.ts

```diff
@@ -135,6 +135,7 @@ export const BuildingSchema = Yup.object().shape({
     .min(0, 'Floor Count must be a valid number')
     .transform(emptyStringToNull),
   buildingTenancy: Yup.string().max(100, 'Tenancy must be less then 100 characters'),
+  buildingTenancyUpdatedOn: Yup.string().required('Required'),
```

Comment:
updatedOn date is required but the tenancy field is not required?

Reply:
right, good catch, need this to be required only when the tenancy is specified.